ethereum/referencetests/src/reference-test/java/org/hyperledger/besu/ethereum/vm/BlockchainReferenceTestTools.java

```diff
@@ -80,7 +80,7 @@ public class BlockchainReferenceTestTools {
     // Inconclusive fork choice rule, since in merge CL should be choosing forks and setting the
     // chain head.
     // Perfectly valid test pre-merge.
-    params.ignore("UncleFromSideChain_Merge");
+    params.ignore("UncleFromSideChain_(Merge|Shanghai|Cancun|Prague|Osaka|Bogota)");
```

Comment:
Damn, wish I had spotted this before trying to debug 😓

Reply:
This is a sneaky one to remember for all future upgrades...maybe can flip it to hardcode pre-merge fork names?

Reply:
I have names encoded for the next 4 releases.  I expect this test will be deleted in 2-3 years.